src/import_edf.jl

```diff
@@ -620,7 +626,7 @@ a signal, its encoded (usually `Int16`) data is copied directly into the
 
 """
 function onda_samples_from_edf_signals(target::SamplesInfoV2, edf_signals,
-                                       edf_seconds_per_record)
+                                       edf_seconds_per_record; dither_storage)
```

Comment:
```suggestion
                                       edf_seconds_per_record; dither_storage=missing)
```

and the associated docstring. otherwise it's technically breaking 😄

Reply:
Addressed in commit [b90e4b7](https://github.com/beacon-biosignals/OndaEDF.jl/pull/72/commits/b90e4b782e9803b33fd48ec16ffda1edc0e10bc7)